qiskit/providers/ibmq/__init__.py

```diff
@@ -97,6 +98,18 @@
 from .runner_result import RunnerResult
 from .version import __version__
 
+# pylint: disable=line-too-long
+warnings.warn(
+    f"The package {__name__} is being deprecated. "
+    "Please see "
+    "https://qiskit.org/documentation/partners/qiskit_ibm_provider/tutorials/Migration_Guide_from_qiskit-ibmq-provider.html"
```

Comment:
I just created ibm.biz/provider_migration_guide, which is shorter and can be updated if the url changes. Maybe a better option?

```suggestion
    "https://ibm.biz/provider_migration_guide"
```

Reply:
Just FYI the original link was added to [the release note](https://github.com/Qiskit/qiskit-terra/pull/8080/files#diff-a7772d1c2d1e096b092f263558880069ac8c4cda7539c3e4851e1f9aef54fbe3R12) of https://github.com/Qiskit/qiskit-terra/pull/8080. If the url changes, we can consider change there as well.

Reply:
But probably the best way to handle the link change would be redirect the original link to the new link. This can be done in sphinx using reredirect extension. Anyway, we don't need to worry about that now.